bin/lib/zopen-download

```diff
@@ -12,13 +12,14 @@ printSyntax()
   echo "If you have a Github OAUTH token, export the environment variable ZOPEN_GIT_OAUTH_TOKEN" >&2
   echo "Syntax: zopen-download [<option>]*" >&2
   echo "  where <option> may be one or more of:" >&2
-  echo "  --list: list all available z/OS Open Tools"  >&2
-  echo "  --update: update z/OS Open Tools packages. On by default."  >&2
-  echo "  --force: force downloads all z/OS Open Tools packages."  >&2
+  echo "  --list: list all available z/OS Open Tools. If no parameters are specified, --list is the default."  >&2
+  echo "  -u|--update|--upgrade: updates installed z/OS Open Tools packages."  >&2
```

Comment:
I know with Brew that upgrade and update mean different things. Can we just go with the one that brew used (I think it is update but not positive)

Reply:
The brew upgrade command is used to upgrade installed packages to the latest version. Same goes for `apt-get upgrade`. `brew update`, `apt-get update` gets the latest package lists and doesn't actually update anything. 

We could introduce a similar option, but currently we always fetch the package info directly from github.

However, as a future improvement, we could cache the package info from github and use `zopen update` to update the cache.

Reply:
So are you thinking just have upgrade for now and we could consider update in the future?

Reply:
Yeah, I'll remove update to avoid confusion